docs/guide/exe_verilator.rst

```diff
@@ -2441,6 +2442,14 @@ The grammar of control commands is as follows:
    following :code:`<signame>` can contain :code:`*` and :code:`?` wildcard
    characters that match any substring or any single character respectively.
 
+.. option:: sc_biguint -module "<modulename>" [-function "<funcname>"] -var "<signame>"
+
+.. option:: sc_biguint -module "<modulename>" [-task "<taskname>"] -var "<signame>"
```

Comment:
I realize below sv_bv also had it, but -function and -task makes no sense for these as it only has any applicability to ports.  It's ok to just update docs here; do sv_bv also.

```suggestion
.. option:: sc_biguint -module "<modulename>" -var "<signame>"
```